ncdiff/src/yang/ncdiff/tests/test_running_config.py

```diff
@@ -2,7 +2,7 @@
 """ Unit tests for the ncdiff cisco-shared package. """
 
 import unittest
-from ncdiff import RunningConfigDiff
+from yang.ncdiff import RunningConfigDiff
```

Comment:
Can you check with Takashi if he intended to use ncdiff vs yang.ncdiff?

Reply:
sorry, it seemed I forgot to update this. it's supposed to be `yang.ncdiff`.